### ISSUE TITLE
Feature Request - Enhancements for improved visual accessibility

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#4874]
One of my team members has visually-triggered migraines and a few elements regarding the way Arctos looks triggers them. She has suggested a couple of ideas for improving this issue that others might have come across. For things we don't currently have control over, perhaps this list could be added to for a future grant proposal that could help us accomplish improvements for accessibility.

1. The overall color combination of bright white background with black lettering and blue hyperlinks in general is challenging for folks who deal with sensitivity to sharp contrast. Is there any control over the background color of the catalog record display page? Is there a possibility for changes at the universal level? The Collection level? The individual user level? In some software options you have the ability to select on each page a range of colors for the background (e.g., light grey, light green, light blue, parchment, etc.).

2. The old data entry background color was a nice calm moss green that was very comforting for those of us who do data entry on the individual item basis. The new data entry page changed to the stark white/black color contrast. Is it possible to go back to the nice green background for data entry? (See above for some other color ideas or options?)
<img width="621" alt="Screen Shot 2022-07-28 at 2 55 13 PM" src="https://user-images.githubusercontent.com/17605945/181650631-43c30d92-3e6b-4a39-8ed6-341fc1a9f666.png">

3. Some users like to use Dark Mode in their browser or their operating system (e.g. Mac dark mode - does Windows have this option?) Arctos does not respond to the dark mode by shifting it's color selection like some websites (e.g. Github - see screenshot below). Is this something we can do?
<img width="2336" alt="Screen Shot 2022-07-28 at 2 51 54 PM" src="https://user-images.githubusercontent.com/17605945/181650370-de85520f-e237-4bf2-98ba-969bc5fd014e.png">

Again, I have no idea what we have control over and what we have to say "new API/visual interface required" but I thought I'd raise the issue since i have a staff member who constantly has to manually adjust the brightness on her computer screen depending on which window she has open. Please feel free to add other issues related to accessibility improvements!

**Comment by dustymc** [comment]
See also https://github.com/ArctosDB/internal/issues/74, https://github.com/ArctosDB/arctos/issues/2893, and I believe @keg34 had a similar situation that I can't find right now.

There is a site-wide CSS file and collections can load their own CSS, but that will almost certainly be an imperfect solution because the code isn't written to fully use CSS. (And maybe that's OK, we can address things as they become a problem, we don't seem to be moving towards something like a full framework very quickly, and maybe we don't need to - CSS and JS have come a long way since that sort of thing became standard-ish. I'm totally up for radical ideas.)

Getting to something like a dark mode option probably involves coming up with some framework or hiring a designer, but I'd not be terribly surprised to learn I know even less about that sort of thing than I think I do (which is "not much") either.

**Comment by AJLinn** [comment]
Thanks for tagging the other issues. I'd be happy to learn how to utilize the CSS option (sorry, I have zero knowledge what that means...).

**Comment by Jegelewicz** [comment]
We really need a grant to hire a web designer.

**Comment by dustymc** [comment]
The data entry screen will have the green background for next release, unless someone has better ideas today-ish. I'm not sure I even notice colors, would be good to find some sort of site-wide palette even if we can't fully implement it.

>utilize the CSS option

That's (mostly, again the lack of a framework is limiting) for your GUID pages, firefox (and probably everything else) has some tools built in, you can inspect and change stuff....

<img width="843" alt="Screen Shot 2022-08-01 at 5 51 37 AM" src="https://user-images.githubusercontent.com/5720791/182151893-8d83fa1d-98d2-484a-86b2-83a8d4dabf54.png">

style.css is the main file, but you can (mostly/probably) control things that aren't in there. From there you can build a file that contains "your" rules, check it into the code (just email it to me), and it will take precedence on at least GUID pages.

Existing customizations are https://github.com/ArctosDB/PG/tree/master/includes/css


>web designer.

Beggars and choosers and etc,, but if I could pick it would be someone a lot more technical - someone who could help us get the framework, whatever form it might take, set up so that things like swapping colors (eg working with a designer) are trivial.

(OK, fine, that's all lies, if it was really up to me most of ya'll would be using custom apps across the Arctos API, but we'd probably always still need a "core" UI and it would ideally be easy to color.)

**Comment by dustymc** [comment]
Closing, please open new issues for any needed UI adjustments.
